package.json

```diff
@@ -41,7 +41,7 @@
     "bottleneck": "1.5.x",
     "event-stream": "3.2.2",
     "express": "4.11.2",
-    "hiredis": "0.4.0",
+    "hiredis": "~0.4.1",
```

Comment:
Yes, please remove the tilde, it seems like they did fix it and we just need to update our `.travis.yml` to tell Travis to install g++-4.8. If you want to add [these lines](https://github.com/redis/hiredis-node/blob/master/.travis.yml#L9-L10) yourself as part of the PR you're welcome to, otherwise I'll do later on when I get a chance.

Also, don't forget to update the warning message (see comment below).